### DISCUSSION TITLE
huh?

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#875] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
3kh0 why the hell did you change the repo name to BIG BALLS REMOVER

**Comment by AshtonDavies** [comment]
I've been trying to remove my balls from my friend's Chromebook for a long time. They're stuck attached by administrator for some reason.

**Reply by S-PScripts** [reply]
💀

**Comment by tuta-amb** [comment]
Maybe because the URL https://github.com/3kh0/ext-remover was blocked for some people? It was for my friend.

**Reply by S-PScripts** [reply]
balls remover will probably be blocked more likely!??!

**Reply by tuta-amb** [reply]
@S-PScripts No? I mean it didn't exist before, why would it be blocked.

**Reply by S-PScripts** [reply]
"balls" might get auto blocked

**Reply by tuta-amb** [reply]
@S-PScripts That is actually ridiculous.

**Reply by catfoolyou** [reply, marked as answer]
He's got a point. Some admins do that kind of shit because they are lazy assholes and dont wanna do their job properly. Getting their salaries off my dad's taxes. Goddammit.
